scripts/dev/templates/operator/Dockerfile.builder

```diff
@@ -0,0 +1,20 @@
+ARG builder_image
+FROM ${builder_image} AS builder
+
+RUN mkdir /workspace
+WORKDIR /workspace
+
+COPY go.mod go.sum ./
+COPY cmd/manager/main.go cmd/manager/main.go
+COPY api/ api/
+COPY controllers/ controllers/
+COPY pkg/ pkg/
+COPY build/bin/ build/bin/
```

Comment:
This is very fragile. Whenever we introduce a new directory, we'll be scratching our heads why it's not included in the image by default.

Here's an alternative way to do it: https://github.com/openshift/cert-manager-operator/blob/master/images/ci/Dockerfile#L2-L3

Reply:
Yes, I know. It's good point. I'll change it. The funny thing is that I just mostly copied the old dockerfile here :)